Packs/GenerateAsBuilt/Scripts/GenerateAsBuilt/usecase_document.html

```diff
@@ -0,0 +1,121 @@
+<head>
```

Comment:
what is this file?

Reply:
Same as other html document, it's easier to edit the styles here and then copy paste into the script. We can remove these from the PR if not suitable.

Reply:
Removed